app/storage/repos/persona.js

```diff
@@ -28,7 +28,8 @@ const formatPersona = (persona) => {
     type: details[1],
     name: persona.name,
     persona: persona.persona,
-    version: persona.version
+    version: persona.version,
+    rowKey: persona.rowKey
```

Comment:
We should rename this to `personaId` just so it's a bit more descriptive and not too tied into the API backend implementation.